dbt_meshify/storage/dbt_project_editors.py

```diff
@@ -14,28 +16,107 @@
 from dbt_meshify.utilities.grouper import ResourceGrouper
 
 
-class DbtSubprojectCreator:
+class YMLOperationType(str, Enum):
+    """ProjectDependencyTypes define how the dependency relationship was defined."""
+
+    Move = "move"
+    Delete = "delete"
```

Comment:
Getting a jump on ChangeSet work, eh 😝 Happy to see this pattern getting some reps in before the refactoring work.